tools/launchdev.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+if [[ "$(uname 2>/dev/null)" == "Darwin" ]]; then
+    VIRTUALENV_DIR=virtualenv-osx
+else
+    VIRTUALENV_DIR=virtualenv
+fi
+VENV=${ST2_REPO}/${VIRTUALENV_DIR}
```

Comment:
Can you set ST2_REPO to ${PWD} if ST2_REPO is not set? Otherwise, I end up with 
```
tools/launchdev.sh: line 223: /virtualenv-osx/bin/activate: No such file or directory
```

Reply:
Good catch!

Done (I think): https://github.com/StackStorm/st2/pull/4145/files#diff-b4e17371e9e3b263027f6a393d3bd772R9